cmd/apiserver/main.go

```diff
@@ -53,31 +53,32 @@ func init() {
 
 // @BasePath  		/api
 func main() {
+	cli.HelpFlag.(*cli.BoolFlag).Usage = "Show help"
```

Comment:
It took me a bit to figure out why this was needed. It's to get `Show` to be capitalized, right? I appreciate your dedication to getting the output consistent!

A comment would be helpful for the future so others know why this was needed.

Reply:
pushed a commit to the branch for this